packages/common/amplify_db_common_dart/pubspec.yaml

```diff
@@ -12,7 +12,7 @@ dependencies:
   amplify_core: ">=1.0.0-next.4 <1.0.0-next.5"
   async: ^2.8.0
   aws_common: ">=0.4.0 <0.5.0"
-  drift: ">=2.3.0 <2.4.0"
+  drift: ^2.4.2
```

Comment:
I think there was a reason we locked this down. 2.4.0 had some breaking changes IIRC. The commit that last updated this probably has context. Do you know if we made changes to handle that, or if drift made changes to make it backwards compatible?